### PR TITLE
Add celery worker cron_queue for scheduled tasks

### PR DESCRIPTION
**Pull request by smigacz**

### CODE REVIEW

**Review by TheMickeyMike — APPROVED** [review]

File: docker/entrypoint-celery-cron-worker.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+exec celery --workdir=/home/app/vaas \
```
Comment (awmackowiak) [inline docker/entrypoint-celery-cron-worker.sh:3]: I wonder, if we will use schedule tasks (cron jobs) why we don't use celery beat for that?

**Review by awmackowiak — APPROVED** [review]

File: docker/entrypoint-celery-cron-worker.sh (hunk 1 above)
Comment (smigacz) [reply docker/entrypoint-celery-cron-worker.sh:3]: we use celery beat, and we use dedicated worker for tasks executed by scheduler. earlier this task was on our default queue ('worker_queue').

Comment (awmackowiak) [reply docker/entrypoint-celery-cron-worker.sh:3]: OK. Thanks for explanation.

**Review by awmackowiak — APPROVED** [review]

File: docker/entrypoint-celery-cron-worker.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  -A vaas.settings worker \
+  --loglevel=DEBUG \
+  --concurrency=1 \
+  -Q cron_queue
```
Comment (ziollek) [inline docker/entrypoint-celery-cron-worker.sh:7]: Why do we spawn workers for each queue? As far as I know, we could run celery multi to spawn all workers in a single container.

File: docs/quick-start/development.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ To check that the build is working, run `docker-compose ps`. This should give yo
 vaas_backend-statuses_1     /wait-for-it.sh uwsgi:3030 ...   Up
```
Comment (ziollek) [inline docs/quick-start/development.md:29]: If we currently gather statuses via celery task I suppose that backend_statuses container is no longer needed.

**Pull request merged**